index.js

```diff
@@ -109,8 +109,23 @@ function lookUpProfile(firstName, prop) {
 
 //Task 6
 
-const reverseString = (str) => str.split("").reverse().join("");
+const reverseString = str =>
+  str
+    .split("")
+    .reverse()
+    .join("");
 
 //Task 7
 
-const randomFraction = () => Math.round(Math.random() * 10);
\ No newline at end of file
+const randomFraction = () => Math.round(Math.random() * 10);
+
+//Task 8
+
+const factorialize = n => {
+    let result = 1;
```

Comment:
Think about the case, when we pass 0 or 1 as a parameter to this function. In this case there is no need to go into `while` loop. Please add here additional condition before going to `while`.